app/src/main/scala/org/alephium/explorer/service/TransactionHistoryService.scala

```diff
@@ -166,79 +167,95 @@ case object TransactionHistoryService extends StrictLogging {
   }
 
   private def processDataForTimeRanges(
-      data: Seq[(TimeStamp, Int, Int, Long)],
+      data: Seq[(TimeStamp, Int, Int, Long, Option[Long])],
       ranges: Seq[(TimeStamp, TimeStamp)],
       intervalType: IntervalType
-  )(implicit gs: GroupSetting): Seq[(TimeStamp, Int, Int, Long, IntervalType)] = {
+  )(implicit gs: GroupSetting): Seq[(TimeStamp, Int, Int, Long, Long, IntervalType)] = {
     ranges.flatMap { case (from, to) =>
       val perChainCount = gs.chainIndexes.map { chainIndex =>
+        val (count, nonCoinbaseCount) = data
+          .filter { case (timestamp, chainFrom, chainTo, _, _) =>
+            timestamp >= from &&
+            timestamp <= to &&
+            chainFrom == chainIndex.from.value &&
+            chainTo == chainIndex.to.value
+          }
+          .foldLeft((0L, 0L)) {
+            case ((currentCount, currentNonCoinbaseCount), (_, _, _, count, nonCoinbaseCount)) =>
+              (currentCount + count, currentNonCoinbaseCount + nonCoinbaseCount.getOrElse(0L))
```

Comment:
we use `nonCoinbaseCount.getOrElse(0L)` as the DB type is Optional, as we introduce it later with a migration.
But during the processing here, we are guarantee to have a value, as four `hourly` the `txsCount` is fetch from the blocks table [here](https://github.com/alephium/explorer-backend/pull/609/files#r1989412943).
For `daily` we use the data stored in `hourly` so it will always have the freshly inserted data.